arkouda/pdarrayclass.py

```diff
@@ -1412,68 +1524,73 @@ class method to return a pdarray attached to the registered name in the arkouda
         return attach_pdarray(user_defined_name)
 
     def _get_grouping_keys(self) -> List[pdarray]:
-        ''' 
+        """
         Private method for generating grouping keys used by GroupBy.
 
         API: this method must be defined by all groupable arrays, and it
         must return a list of arrays that can be (co)argsorted.
-        '''
+        """
         if self.dtype == akbool:
             from arkouda.numeric import cast as akcast
+
             return [akcast(self, akint64)]
         elif self.dtype in (akint64, akuint64):
             # Integral pdarrays are their own grouping keys
             return [self]
         else:
             raise TypeError("Grouping is only supported on numeric data (integral types) and bools.")
 
-#end pdarray class def
-    
+
+# end pdarray class def
+
 # creates pdarray object
 #   only after:
 #       all values have been checked by python module and...
 #       server has created pdarray already before this is called
 #       server has created pdarray already befroe this is called
 @typechecked
-def create_pdarray(repMsg : str) -> pdarray:
+def create_pdarray(repMsg: str) -> pdarray:
     """
-    Return a pdarray instance pointing to an array created by the arkouda server.
-    The user should not call this function directly.
+        Return a pdarray instance pointing to an array created by the arkouda server.
+        The user should not call this function directly.
 
-    Parameters
-    ----------
-    repMsg : str
-        space-delimited string containing the pdarray name, datatype, size
-        dimension, shape,and itemsize
+        Parameters
+        ----------
+        repMsg : str
+            space-delimited string containing the pdarray name, datatype, size
+            dimension, shape,and itemsize
 
-    Returns
-    -------
-    pdarray
-        A pdarray with the same attributes and data as the pdarray; on GPU
+        Returns
+        -------
+        pdarray
+            A pdarray with the same attributes and data as the pdarray; on GPU
 
-    Raises
--   -----
-    ValueError
-        If there's an error in parsing the repMsg parameter into the six 
-        values needed to create the pdarray instance
-    RuntimeError
-        Raised if a server-side error is thrown in the process of creating
-        the pdarray instance
+        Raises
+    -   -----
+        ValueError
+            If there's an error in parsing the repMsg parameter into the six
+            values needed to create the pdarray instance
+        RuntimeError
+            Raised if a server-side error is thrown in the process of creating
+            the pdarray instance
```

Comment:
Another docstring with extra tab indentation.

Reply:
same issue with the stray `-`, will fix it!